packages/plugins/dashboard-time-track-angular-ui/src/lib/components/time-tracking/time-tracking.component.html

```
@@ -46,10 +25,29 @@ <h4>
 					(click)="logs$.next(true)"
 				>
 					<nb-icon icon="sync-outline"></nb-icon>
-					{{ 'BUTTONS.REFRESH' | translate }}
+					<span class="action-label">{{ 'BUTTONS.REFRESH' | translate }}</span>
+				</button>
+				<ga-timezone-filter
+					(timeFormatChange)="timeFormatChanged($event)"
+					(timeZoneChange)="timeZoneChanged($event)"
+				></ga-timezone-filter>
+				<button
+					class="manage-widget"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)